### PR TITLE
Added option to show bridge location

### PR DESCRIPTION
**Pull request by evfool**
Thanks for this wonderful tool. Would you care to accept an option to show/mark the bridge location too? This PR does just that. If not, I'll deploy my fork somewhere.

### CODE REVIEW

File: fretboard-generator.js [hunk 1]
```diff
@@ -12,11 +12,15 @@ var makerjs = require('makerjs');
  * @param {number} frets - How many frets to calculate for.
  * @return {[number]} A list of positions in the same units as the scale length was provided.
  */
-function generateFretPositions(scaleLength, frets) {
+function generateFretPositions(scaleLength, frets, bridgeLocation) {
```
Comment (mdales) [inline fretboard-generator.js:16]: Can you please also update the document comments above to include the new parameter?

**Review by mdales — COMMENTED** [review]
Thanks for this, it's very good!

I've made one very trivial comment - if you can fix that then i'll happily merge this!

**Comment by mdales** [comment]
Many thanks again for this! I'll update my deployment on the website tonight (don't have my S3 creds to hand just now), and I'll give you a shout out in my next blog update, as this is a useful new addition!

**Comment by mdales** [comment]
I've just pushed the update live to where I host the tool, thanks again! If you have a real name and/or website where I can point people, let me know, as I'll mention them in my weekly blog post.

**Comment by evfool** [comment]
My name is Robert Roth, you can use that (or simply Robert). I don't have a real page (yet), only one on FB, called Chol Guitars (https://facebook.com/cholguitars) but there's not a lot to share there, I'm just starting out :) But hey, everyone has to start somewhere, and creating some tools is a must. Keep up the good work!

**Pull request merged**